alibi_detect/saving/_tensorflow/saving.py

```diff
@@ -158,7 +158,7 @@ def save_embedding_config(embed: TransformerEmbedding,
     return cfg_embed
 
 
-def save_optimizer_config(optimizer: tf.keras.optimizers.Optimizer):
+def save_optimizer_config(optimizer: Union[tf.keras.optimizers.Optimizer]):
```

Comment:
Unnecessary `Union`?

Reply:
Changed to `Union[tf.keras.optimizers.Optimizer, tf.keras.optimizers.legacy.Optimizer]` in https://github.com/SeldonIO/alibi-detect/pull/739/commits/a3f0e90e593420c391a1638aa562f67ce869ce5d